crates/uv/tests/it/export.rs

```diff
@@ -1174,6 +1176,122 @@ fn requirements_txt_https_git_credentials() -> Result<()> {
     Ok(())
 }
 
+/// SSH blocks too permissive key files.
+fn reduce_key_permissions(key_file: &Path) -> Result<()> {
+    #[cfg(unix)]
+    {
+        use std::fs::Permissions;
+        use std::os::unix::fs::PermissionsExt;
+        fs_err::set_permissions(key_file, Permissions::from_mode(0o400))?;
+    }
+    #[cfg(windows)]
+    {
+        use std::process::Command;
+
+        // https://superuser.com/a/1489152
+        Command::new("icacls")
```

Comment:
Minor note this isn't tested in CI because we skip the `git` feature

Reply:
Yeah, this is only for local Windows development. We can iterate over the command if it fails for specific Windows setups.